test/model/message_test.dart

```diff
@@ -123,6 +127,179 @@ void main() {
     });
   });
 
+  group('edit-message methods', () {
+    late PerAccountStore store;
+    late FakeApiConnection connection;
+    late StreamMessage message;
+
+    Future<void> prepare() async {
+      store = eg.store();
+      connection = store.connection as FakeApiConnection;
+      message = eg.streamMessage();
+      await store.addMessage(message);
```

Comment:
I think we can also test if the listeners are notified whenever an update is expected.